specs/prometheus-nodeexporter/SOURCES/prometheus-nodeexporter.service

```diff
@@ -0,0 +1,19 @@
+# -*- mode: conf -*-
+
+[Unit]
+Description=Prometheus Node Exporter
+Documentation=https://github.com/prometheus/node_exporter
+After=network.target
+
+[Service]
+EnvironmentFile=-/etc/sysconfig/prometheus-nodeexporter
+User=nodeexporter
+StandardOutput=/var/log/prometheus-nodeexporter/prometheus-nodeexporter.log
```

Comment:
@gongled FYI you can't do things like that. Only since 236 (_219 on CentOS 7 for now_) you can write something like
`StandardOutput=file:/var/log/file.log`. With 219 you can't redirect stdout/stderr to the file.

Same problem in prometheus spec.